instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -7,13 +7,16 @@ package otelgin // import "go.opentelemetry.io/contrib/instrumentation/github.co
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/gin-gonic/gin"
 
+	internalsemconv "go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin/internal/semconv"
 	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin/internal/semconvutil"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/metric"
 	"go.opentelemetry.io/otel/propagation"
 	semconv "go.opentelemetry.io/otel/semconv/v1.20.0"
```

Comment:
For another PR: we should stop using semconv directly here. Everything we use it for can be replaces with the internal semconv package.

Reply:
You mean this package, `semconv "go.opentelemetry.io/otel/semconv/v1.20.0"`?

Reply:
OK, I understand. Another PR will be provided later.